linux_vm_extras/metadata.csv

```diff
@@ -0,0 +1 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
```

Comment:
@cory-stripe can you fill in the metadata? It should be pretty easy since you already know what the metrics will be. You can look at the examples in integrations core, like mysql, if you need some help: https://github.com/DataDog/integrations-core/blob/master/mysql/metadata.csv

And we'll be available to help out if you need it!